pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/ast/SafeNavigationOperator.txt

```diff
@@ -1,99 +1,91 @@
-      +- Field[@DefiningType = "Foo", @Image = "x", @Name = "x", @Namespace = "", @RealLoc = true, @Type = "Integer", @Value = null]
-      |  +- ModifierNode[@Abstract = false, @DefiningType = "Foo", @DeprecatedTestMethod = false, @Final = false, @Global = false, @InheritedSharing = false, @Modifiers = 0, @Namespace = "", @Override = false, @Private = false, @Protected = false, @Public = false, @RealLoc = true, @Static = false, @Test = false, @TestOrTestSetup = false, @Transient = false, @Virtual = false, @WebService = false, @WithSharing = false, @WithoutSharing = false]
-      +- Field[@DefiningType = "Foo", @Image = "profileUrl", @Name = "profileUrl", @Namespace = "", @RealLoc = true, @Type = "String", @Value = null]
-      |  +- ModifierNode[@Abstract = false, @DefiningType = "Foo", @DeprecatedTestMethod = false, @Final = false, @Global = false, @InheritedSharing = false, @Modifiers = 0, @Namespace = "", @Override = false, @Private = false, @Protected = false, @Public = false, @RealLoc = true, @Static = false, @Test = false, @TestOrTestSetup = false, @Transient = false, @Virtual = false, @WebService = false, @WithSharing = false, @WithoutSharing = false]
-      +- FieldDeclarationStatements[@DefiningType = "Foo", @Namespace = "", @RealLoc = true, @TypeName = "Integer"]
-      |  +- ModifierNode[@Abstract = false, @DefiningType = "Foo", @DeprecatedTestMethod = false, @Final = false, @Global = false, @InheritedSharing = false, @Modifiers = 0, @Namespace = "", @Override = false, @Private = false, @Protected = false, @Public = false, @RealLoc = false, @Static = false, @Test = false, @TestOrTestSetup = false, @Transient = false, @Virtual = false, @WebService = false, @WithSharing = false, @WithoutSharing = false]
-      |  +- FieldDeclaration[@DefiningType = "Foo", @Image = "anIntegerField", @Name = "anIntegerField", @Namespace = "", @RealLoc = true]
-      |     +- VariableExpression[@DefiningType = "Foo", @Image = "anIntegerField", @Namespace = "", @RealLoc = true]
-      |     |  +- ReferenceExpression[@Context = null, @DefiningType = "Foo", @Namespace = "", @RealLoc = false, @ReferenceType = ReferenceType.LOAD, @SObjectType = false, @SafeNav = true]
-      |     |     +- VariableExpression[@DefiningType = "Foo", @Image = "anObject", @Namespace = "", @RealLoc = true]
++- ApexFile[@DefiningType = null, @Namespace = null, @RealLoc = true]
+   +- UserClass[@DefiningType = "c__Foo", @Image = "Foo", @Namespace = null, @RealLoc = true, @SimpleName = "Foo", @SuperClassName = ""]
+      +- ModifierNode[@Abstract = false, @DefiningType = "c__Foo", @DeprecatedTestMethod = false, @Final = false, @Global = false, @InheritedSharing = false, @Modifiers = 1, @Namespace = null, @Override = false, @Private = false, @Protected = false, @Public = true, @RealLoc = false, @Static = false, @Test = false, @TestOrTestSetup = false, @Transient = false, @Virtual = false, @WebService = false, @WithSharing = false, @WithoutSharing = false]
```

Comment:
ModifierNode/@<!-- -->RealLoc is false now...
The same with Field/@<!-- -->RealLoc

Reply:
Done.